src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robotArr = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0, y: 0,
+    },
+    get info() {
+      return `name: ${
+        this.name
+      }, chip version: ${
+        this.version}, wheels: ${this.wheels}`;
```

Comment:
And also destructure values form `this`
```suggestion
      return `name: ${this.name}, chip version: ${
        this.version}, wheels: ${this.wheels}`;
```